plugin/emacs/flymake-quicklintjs.el

```diff
@@ -0,0 +1,138 @@
+;;; flymake-quicklintjs.el ---  Flymake support for quick-lint-js -*- lexical-binding: t; -*-
```

Comment:
I see why people prefer Flycheck over Flymake... You sure have to do a lot of work yourself.